DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/AdminUserDataServiceTests.cs

```diff
@@ -53,6 +53,20 @@ public void GetAdminById_returns_admin_with_correct_user_centre_details()
             returnedAdminEntity.UserCentreDetails.EmailVerified.Should().Be(expectedUserCentreDetails.EmailVerified);
         }
 
+        [Test]
+        public void GetAdminsByCentreId_Returns_admin_list()
+        {
+            // Given
+            var expectedAdminEntity = UserTestHelper.GetDefaultAdminEntity();
+
+            // When
+            var returnedAdmins =
+                userDataService.GetAdminsByCentreId(expectedAdminEntity.AdminAccount.CentreId).ToList();
+
+            // Then
+            returnedAdmins[0].Should().BeEquivalentTo(expectedAdminEntity);

```

Comment:
Since `returnedAdmins` is a list with multiple entries, this should instead test that `returnedAdmins` contains `expectedAdminEntity`, and that all entries in `returnedAdmins` have a centre id of `expectedAdminEntity.AdminAccount.CentreId`. 

If `GetAdminsByCentreId` is changed to only return active records (see other comment), there should also be a test that no inactive records are returned.

(Don't forget to use an AssertionScope to wrap multiple assertions.)